tests/unit/tests/PaymentGateways/Gateways/Stripe/CreditCardGatewayTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+use Give\Framework\PaymentGateways\Commands\PaymentProcessing;
+use Give\Framework\PaymentGateways\Commands\RedirectOffsite;
+use Give\PaymentGateways\DataTransferObjects\GatewayPaymentData;
+use Give\PaymentGateways\Gateways\Stripe\CreditCardGateway;
+
+/**
+ * @unreleased
+ */
+class CreditCardGatewayTest extends Give_Unit_Test_Case
+{
+    /** @test */
+    public function it_creates_a_payment_that_is_processing()
```

Comment:
@kjohnson can we follow the [camel case ](https://en.wikipedia.org/wiki/Camel_case)of the function name to main train consistency?

Reply:
I'm happy to make changes to match the style guide. I would like to discuss [snake_case](https://en.wikipedia.org/wiki/Snake_case) for readability in tests - the methods aren't a public API, so I'm less worried about matching camel case of the codebase. I'll admit that this is a carry over from also working in Laravel.

Reply:
A big difference being that test methods are sentences, while regular methods are names.